.github/workflows/build-triton-wheel.yml

```diff
@@ -65,6 +65,8 @@ jobs:
           docker-image: ${{ env.DOCKER_IMAGE }}
 
       - name: Build Triton wheel
+        env:
+          IS_RELEASE_TAG: ${{ startsWith(github.event.ref, 'refs/tags/v') }}
```

Comment:
It feels like this is something that should have been landed in main first, and than cherry-picked into release

Reply:
Its part of the revert, we actually first RC from pin rather then branch since branch was not ready https://github.com/pytorch/pytorch/commit/4a3dabd67f8ce63f2fc45f278421cca3cc532cfe